### DISCUSSION TITLE
CX410: IPC_NT1NA44MP.2207_23051803.CX410.OS04A10.4MP.REOLINK

### DISCUSSION THREAD

**Discussion by mblaser79** [AT0myks/reolink-fw-archive#30] [Firmware sharing]
https://drive.google.com/file/d/1Ta6vwhegTdYJImILwUtKeJVtWCVhKrhb/view?usp=sharing

This is the last version of the CX410 firmware that allows its light to be off at all times and then come on when motion is detected. Subsequent firmware updates require the lights be on at all times (if you don't have what Reolink considers to be enough ambient light). See: https://www.reddit.com/r/reolinkcam/comments/16u1ppy/is_there_a_way_to_turn_off_the_cx410_warm_light/

**Comment by AT0myks** [comment]
Thanks, added in 71bc729f95ed5472a6efc9a138d2412820bfb123.
Reading the post you linked it seems you've confirmed that v3.1.0.2272_23053066 has this issue, but is it also the case for v3.1.0.2501_23072503?

**Reply by mblaser79** [reply]
Correct, .2501 also acts the same way.

By the way, I realized I also have another fw that you don't have, v3.1.0.2447_23071263. It was sent to me by Reolink support after I asked for the update Willson1_ was talking about in [this post](https://www.reddit.com/r/reolinkcam/comments/15aox6o/cx410_spotlight_audio_questions/). It didn't change anything though, it still acts the same way. I don't know what else is different about it.

I have it on my Google drive if you want it for the archive: https://drive.google.com/drive/folders/1TGwHlBxXzNY4sZTvGtSuIutPVnL9brCw?usp=sharing

**Reply by AT0myks** [reply]
This one and two others, so that's [3 more](https://github.com/AT0myks/reolink-fw-archive/commit/941c975253d7f87da4361178b11c72d5badf773f). Thanks for sharing the link to your archive.
For the CX410, I changed the note of v3.1.0.2207_23051803 in c00b359208ad09301e4dedf534990159030aa654 to make it clearer that it has light settings that the other firmwares don't. If that's not really accurate please tell me and I'll change it. At least the source points here so that users can read your post.

**Comment by Eric-ST** [comment]
Hello. Please can you repost the firmware again? My CX410 keeps the lights on even during smart detection after a firmware update and I want to attempt to fix it with the one you have.

**Reply by Eric-ST** [reply]
I found it. Thanks.
